src/scss/components/optional/_loading-indicator.scss

```diff
@@ -0,0 +1,50 @@
+/**
+* Components > Spinner Indicator
+* =================================
+*
+* Example: f-searchbox -  Loqate feature enabled (Full address capture)
+*
+* If you'd like to use it in your project you can include it by adding `@include loadingIndicator();` into your SCSS dependencies file.
+*
+*/
+
+$loading-indicator-size-medium: 20px;
+$loading-indicator-size-large: 48px;
```

Comment:
Reviewing my own code here. But I am open to just passing in the pixel size to this mixin but thought we might want set sizes?

Reply:
I personally think set sizes would be best. Usually these are set via the UI designer. Otherwise we'd probably end up with lots of sizes being used. Happy to go with what everyone else things tho.

Reply:
Yeah, it definitely seems like the better approach. My main gripe was naming convention just in case we get 5+ sizes, but I think I might be looking a bit too deep into it!

Reply:
Yeah that will most likely happen. Think that would be something we could look at later. Sometimes tho i've often found that having say 4 sizes already defined makes it easier on pushing back on extra sizes coming in :)